package.json

```diff
@@ -38,8 +38,8 @@
     "build-demo-app": "npm run build; cd packages/browser/examples/demoClientApp/; npm ci; npm run build; mkdir -p ../../../../dist/website/demo/; cp -r dist/. ../../../../dist/website/demo/"
   },
   "devDependencies": {
-    "@inrupt/eslint-config-base": "^2.0.4",
-    "@inrupt/eslint-config-lib": "^2.1.2",
+    "@inrupt/eslint-config-base": "^2.1.3",
```

Comment:
We should be able to get rid of this `@inrupt/eslint-config-base` dependency entirely as it is already a dependency of `@inrupt/eslint-config-lib`

Reply:
Fixed in https://github.com/inrupt/solid-client-authn-js/pull/3100/commits/cdf7b4252cd52a74d5bcb3f2948f6939f931f119